docs/app/app.md

```diff
@@ -10,47 +10,7 @@ This document defines events and attributes related to client-side applications
-| [`app.screen.coordinate.x`](/docs/attributes-registry/.md) | int | The x (horizontal) coordinate of a screen coordinate, in screen pixels. | `0`; `131` | `Recommended` | ![Development](https://img.shields.io/badge/-development-blue) |
-| [`app.screen.coordinate.y`](/docs/attributes-registry/.md) | int | The y (vertical) component of a screen coordinate, in screen pixels. | `12`; `99` | `Recommended` | ![Development](https://img.shields.io/badge/-development-blue) |
-| [`app.widget.id`](/docs/attributes-registry/.md) | string | An identifier that uniquely differentiates this widget from other widgets in the same application. [2] | `f9bc787d-ff05-48ad-90e1-fca1d46130b3`; `submit_order_1829` | `Recommended` | ![Development](https://img.shields.io/badge/-development-blue) |
-| [`app.widget.name`](/docs/attributes-registry/.md) | string | The name of an application widget. [3] | `submit`; `attack`; `Clear Cart` | `Recommended` | ![Development](https://img.shields.io/badge/-development-blue) |
-
-**[1] `app.installation.id`:** Its value SHOULD persist across launches of the same application installation, including through application upgrades.
-It SHOULD change if the application is uninstalled or if all applications of the vendor are uninstalled.
-Additionally, users might be able to reset this value (e.g. by clearing application data).
-If an app is installed multiple times on the same device (e.g. in different accounts on Android), each `app.installation.id` SHOULD have a different value.
-If multiple OpenTelemetry SDKs are used within the same application, they SHOULD use the same value for `app.installation.id`.
-Hardware IDs (e.g. serial number, IMEI, MAC address) MUST NOT be used as the `app.installation.id`.
-
-For iOS, this value SHOULD be equal to the [vendor identifier](https://developer.apple.com/documentation/uikit/uidevice/identifierforvendor).
-
-For Android, examples of `app.installation.id` implementations include:
-
-- [Firebase Installation ID](https://firebase.google.com/docs/projects/manage-installations).
-- A globally unique UUID which is persisted across sessions in your application.
-- [App set ID](https://developer.android.com/identity/app-set-id).
-- [`Settings.getString(Settings.Secure.ANDROID_ID)`](https://developer.android.com/reference/android/provider/Settings.Secure#ANDROID_ID).
-
-More information about Android identifier best practices can be found [here](https://developer.android.com/training/articles/user-data-ids).
-
-**[2] `app.widget.id`:** A widget is an application component, typically an on-screen visual GUI element.
-
-**[3] `app.widget.name`:** A widget is an application component, typically an on-screen visual GUI element.
-
-<!-- markdownlint-restore -->
-<!-- prettier-ignore-end -->
-<!-- END AUTOGENERATED TEXT -->
-<!-- endsemconv -->
+See the [app attributes](/docs/attributes-registry/app.md) registry for details. 
```

Comment:
not a strong opinion, but I'd rather not even reference all possible app attributes here. We consider attribute registry as a giant pool of might-be-related things and try to de-prioritize it a bit comparing to the telemetry signals.

If you feel super=strong about keeping it, I'd suggest something like

 ```suggestion
See the [app attributes](/docs/attributes-registry/app.md) registry for all application-related attributes that may appear on telemetry items.
```

I'd also move this section down, to be after all event definitions